symfony/plugins/orangehrmAdminPlugin/Api/LocationAPI.php

```diff
@@ -0,0 +1,180 @@
+<?php
+/**
+ * OrangeHRM Enterprise is a closed sourced comprehensive Human Resource Management (HRM)
```

Comment:
pls remove the `Enterprise` header .. we are open source :)

Reply:
Oops, never saw it coming.. :) done!